offlinevalidator/offlinevalidator.go

```diff
@@ -448,6 +412,84 @@ func sliceContains(slice []string, match string) bool {
 	return false
 }
 
+func mapContainsNil(mp map[string]interface{}) bool {
+	for _, m := range mp {
+		if m == nil {
+			return true
+		}
+	}
+	return false
+}
+
+func sliceContainsNil(slice []interface{}) bool {
```

Comment:
I would also see that and `mapContainsNil` and others generic ones in this module as helpers function.